packages/orchestration/src/orchestration-response.ts

```diff
@@ -51,6 +55,54 @@ export class OrchestrationResponse {
     return choice?.message?.content;
   }
 
+  /**
+   * Messages that can be used for subsequent prompts as message history.
+   * @param choiceIndex - The index of the choice to parse.
+   * @returns A list of all messages.
+   */
+  getAllMessages(choiceIndex = 0): ChatMessages {
+    const messages = (this.data.module_results.templating ?? []).map(
+      (message: ChatMessage) => {
+        const isMultiContent = Array.isArray(
```

Comment:
[req] please consider the naming here. `isMultiChatMessage` is more speaking for what you're checking and subsequent function calls are also named accoridngly